standard.md

```diff
@@ -0,0 +1,136 @@
+# Definition of the standard
+
+This document intends to define a standard
+with the purpose of facilitating the sharing of source codes used to
+analyze climate data and produce evaluation results and figures. 
+Said codes will hereafter be referred as 'diagnostic scripts', or in short 'diagnostics'
+or 'scripts'. 
+
+It is assumed that diagnostic scripts are able to be wrapped
+as standalone codes that run as independent processes, an approach that
+has been successfully tested in tools such as ESMValTool and CLiMAF.
+It also assumes that diagnostic scripts are able to read CF compliant NetCDF
+data files and to produce only the same kind of output files and graphic files.
```

Comment:
```suggestion
data files.
```
I have no clue what that means, do you?

Reply:
It means :

> data files and to produce only NetCDF and graphic output files.

Reply:
In the ESMValTool we have several diagnostics that produce other output: CSV files, text files, Excel spreadsheets, etc. If this is what scientists would like their diagnostic to produce, why not let them?

Reply:
Fine. Should then state that any output kind is allowed.